scripts/verify_firmware_image.py

```
@@ -0,0 +1,219 @@
+"""
+
+import csv
+import hashlib
+import os
+import struct
+import sys
+
+try:
+    Import("env")  # noqa: F821 — provided by SCons in PlatformIO post-build context
+    _PIO_CONTEXT = True
+except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)